Makefile

```diff
@@ -1718,7 +1718,7 @@ install_qc: install $(HEXAGON_RUNTIME_LIBS)
 # we include in our distribution.
 $(BUILD_DIR)/halide_config.%: $(ROOT_DIR)/tools/halide_config.%.tpl
 	@mkdir -p $(@D)
-	cat $< | sed -e 's/$${LLVM_SYSTEM_LIBS}/${LLVM_SYSTEM_LIBS}/g' > $@
+	cat $< | sed -e 's/@HALIDE_SYSTEM_LIBS_RAW@/${LLVM_SYSTEM_LIBS}/g' > $@
```

Comment:
Why is this HALIDE_SYSTEM_LIBS_RAW instead of HALIDE_SYSTEM_LIBS?

Reply:
`HALIDE_SYSTEM_LIBS_RAW` is the new value that is to be replaced in `tools/halide_config.bzl.tpl` and `tools/halide_config.cmake.tpl`. In CMake, the value will be automatically replaced by the value of the variable (within the CMake configuration), but with Makefiles, we have more flexibility through `sed`.